nomad/structs/host_volumes.go

```diff
@@ -429,6 +429,15 @@ type HostVolumeListResponse struct {
 	QueryMeta
 }
 
+type TaskGroupVolumeClaimListRequest struct {
+	QueryOptions
+}
```

Comment:
As I noted in my comment on the routes, I suspect it would be a lot more useful to end users to be able to filter by job ID, task group, volume name, etc. here. Although you can always build a filter with a bexpr expression, that's a poor UX.

Reply:
[21c8351](https://github.com/hashicorp/nomad/pull/25114/commits/21c8351c169557ffc1b8440b038c743e37098987)